index.js

```diff
@@ -102,6 +110,25 @@ route.post('/reset/calls', (_req, res) => {
   res.sendStatus(204);
 });
 
+route.post('/record', (req, res) => {
```

Comment:
proposal. just one route with two methods:

POST /recording updates the full recording context, including recordings that you'd set with /load-recordings
GET /recording returns the full recording context, including recordings